tests/sentry/tasks/test_commit_context.py

```diff
@@ -551,9 +620,23 @@ def test_gh_comment_create_queued(self, mock_comment_workflow):
             assert mock_comment_workflow.called
 
     @with_feature("organizations:pr-comment-bot")
-    def test_gh_comment_update_queue(self, mock_comment_workflow):
+    @patch("sentry.integrations.github.client.get_jwt", return_value=b"jwt_token_1")
+    @responses.activate
+    def test_gh_comment_update_queue(self, get_jwt, mock_comment_workflow):
         """Task queued if new issue for prior comment"""
 
+        responses.add(
```

Comment:
There's two cases where we add identical responses twice - is it possible to factor these out via helper functions, or is there a reason they're hardcoded like this?

Reply:
🤷 helper could be good, although if it's only for a few tests i'm humdrum

Reply:
it IS actually repeated 4x so i pulled it out -- thanks!